iOS Client/DevotionsController/MBDevotionsViewController.swift

```diff
@@ -12,11 +12,14 @@ import ReSwift
 class MBDevotionsViewController: UIViewController, StoreSubscriber, UITableViewDelegate, UITableViewDataSource {
 
     @IBOutlet weak var tableView: UITableView!
-    var devotions: [MBDevotion] = []
+    var devotions: [LoadedDevotion] = []
     override func viewDidLoad() {
         super.viewDidLoad()
         tableView.delegate = self
         tableView.dataSource = self
+        tableView.rowHeight = UITableViewAutomaticDimension
+        
+        tableView.register(UINib(nibName: "DevotionTableViewCell", bundle: nil), forCellReuseIdentifier: "DevotionTableViewCell")
```

Comment:
it would be good to make a constant called `reuseIdentifier` as a private property in this class

Reply:
and use it here and in tableView cellForRowAt